security/access_control.rst

```diff
@@ -265,7 +280,8 @@ key:
             access_control:
                 -
                     path: ^/_internal/secure
-                    allow_if: "'127.0.0.1' == request.getClientIp() or has_role('ROLE_ADMIN')"
+                    roles: 'ROLE_ADMIN'
+                    allow_if: "'127.0.0.1' == request.getClientIp() or request.header.has('X-Secure-Access')"
```

Comment:
you should document with a comment here that the logic is or. You explain it more below, but a comment is all most people will read ;)

Reply:
Done, thank you!